sig/staccato.rbs

```diff
@@ -0,0 +1,367 @@
+module Staccato
```

Comment:
Is this file required for rubocop or something?

Reply:
No, it isn't.  It should be usable by the `steep`, `rbs`, and `typeprof` gems, but I confess that I don't have much experience with rbs at the moment.

`bundle gem staccato` created the file, and I used `typeprof` to automatically create its contents.  IMO, if

Reply:
IMO, it would be much nicer if bundler's project template also created some rake tasks for both _checking_ types and regenerating the RBS.  Since I don't have any type-checking in the CI build, I'm inclined to just delete this file: otherwise it will get out of sync with the code.  What do you think?

Reply:
I do like the idea of the type checking, but if you're okay with it, let's punt on this for this PR. Thanks!

Reply:
👍🏻 I'll delete this file.